### PR TITLE
neetoui major version migration

### LINKED ISSUE #732: Normalize css repeat itself multiple times
Across all **neetoProducts** I can see normalize CSS repeat itself multiple times. Please go through [this](https://www.loom.com/share/e19f078bcc4247e3a967c0c3ed1bfc68) video for more info.

cc @sandip-mane @yedhink @unnitallman

### PR DESCRIPTION
**Pull request by edwinbbu**
Fixes #732 

- neetoui migrated to 3.0.1
- notes table height fixed using ref

Screenshot
Screenshot 2021-11-26 at 11 36 21 AM

### CODE REVIEW

**Comment by edwinbbu** [comment]
@yedhink _a please review

**Comment by edwinbbu** [comment]
cc @karthiknmenon

File: app/javascript/stylesheets/common.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.notes-table-height {
+  height: calc(100vh - 160px);
+}
```
Comment (yedhink) [inline app/javascript/stylesheets/common.scss:3]: Let's follow https://www.bigbinary.com/learn-rubyonrails-book/creating-reusable-components#7-in-1-sass-pattern

This should be within a `components` stylesheet folder with the name `_notes.scss`.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 2]
```diff
@@ -26,10 +26,10 @@ export default function NoteTable({
 }) {
   const [showEditNote, setShowEditNote] = useState(false);
   const [selectedNote, setSelectedNote] = useState({});
-
+  const parentContainerRef = useRef(null);
   return (
     <>
-      <div className="w-full">
+      <div className="w-full notes-table-height" ref={parentContainerRef}>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:32]: I am failing to understand how this `ref` is being useful here. Are we manipulating this `parentContainerRef` anywhere else within this component? What is the difference if we don't pass in this `ref` over here?

**Review by yedhink — COMMENTED** [review]
@edwinbbu _a

I have added some comments. Please go through them and let me know if you need any clarifications.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx (hunk 2 above)
Comment (edwinbbu) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:32]: It is working without the ref. I'll remove it. Thanks.

**Comment by edwinbbu** [comment]
@yedhink _a please check.

**Pull request merged**